finders/schemas/asylum-support-decisions.json

```diff
@@ -266,18 +286,25 @@
       ]
     },
     {
+      "key": "tribunal_decision_landmark_name",
+      "name": "Landmark name",
+      "type": "text",
+      "display_as_result_metadata": true,
+      "filterable": false
+    },
+    {
       "key": "tribunal_decision_reference_number",
       "name": "Reference number",
       "type": "text",
-      "preposition": null,
       "display_as_result_metadata": true,
       "filterable": false
     },
     {
       "key": "tribunal_decision_decision_date",
       "name": "Decision date",
+      "short_name": "Decided",
       "type": "date",
-      "preposition": null,
+      "preposition": "was decided", 
```

Comment:
Good spot. Changing it to `decided`. Then frontend search results appear like this:

1 decision **decided** between 27 April 2006 or 2 April 2006
6 decisions **decided** after 27 April 2006

Aside: noticed finder-frontend shows "27 April 2006 *or* 28 April 2006" when it should be "27 April 2006 *and* 28 April 2006".